fairlearn/experimental/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
```

Comment:
Can we add a short description of the module to indicate that people shouldn't rely on contents of this module and that anything can break from version to version without much warning?